libs/platform/src/lib/components/list/public_api.ts

```diff
@@ -0,0 +1,3 @@
+export * from './list.module';
+export * from './base-list-item';
+export * from './standard-list-item/standard-list-item.module';
```

Comment:
Add `list.component` and `list.config` there as well

Reply:
done